doc/all.mk

```diff
@@ -161,8 +161,8 @@ doc/%.pdf: doc/%.md
 		-V papersize=letter \
 		--template=./scripts/asciidoc/freeradius.template -o $@ $<
 
-.PHONY: asciidoc html pdf clean clean.doc
-asciidoc: $(ADOC_FILES)
+.PHONY: adoc html pdf clean clean.doc
+adoc: $(ADOC_FILES)
```

Comment:
This should just be left as "asciidoc".  The file contents are "asciidoc", not "adoc".  We just use the "adoc" extension because having 100 files named "...asciidoc" is annoying.

Reply:
done